docs/source/resources/support_matrix.rst

```diff
@@ -173,7 +173,9 @@ The following tables provide the software and model versions that NVTabular vers
 | Container Operating System                          | Ubuntu version 20.04                                                   | 
 +-----------------------------------------------------+------------------------------------------------------------------------+
 | Base Container                                      | `nvcr.io/nvidia/tensorflow:21.07-tf2-py3                               |
-|                                                     | <https://nvcr.io/nvidia/tensorflow:21.06-tf2-py3>`_                    |
+|                                                     | <https://nvcr.io/nvidia/tensorflow:21.07-tf2-py3>`_                    |
+|                                                     |                                                                        |
+|                                                     | *Customized with TensorFlow version 2.4.2                              |
```

Comment:
We're getting a failure like 
```/var/jenkins_home/workspace/nvtabular_tests/nvtabular/docs/source/resources/support_matrix.rst:179:Inline emphasis start-string without end-string.
```

I think we need to escape the * character so RST doesn't thing we're trying to add emphasis:


```suggestion
|                                                     | \*Customized with TensorFlow version 2.4.2                              |
```